### ISSUE TITLE
Port number keeps changes on every start up  

### ISSUE THREAD

**Issue by osro** [pantheon-systems/localdev-issues#17]
Could you make it so that the port number of the localdev could be set as constant?

Now it keeps switching between 444 and 4443 on every start up. I have another Yarn job rebuilding changes on my Wordpress theme and it's proxying the localdev address. So I have to restart the server twice to get the localdev port to match my settings.

**Comment by joshkoenig** [comment]
Hey Orso - this usually indicates that you have another local dev tool that's occupying the normal port. Is that possibly the case?

**Comment by osro** [comment]
Nope. I only have yarn job proxying the localdev on port 8005. And this happens even if I don't have this running. 

Just by stopping and restarting the localdev it keeps switching between 444 and 4443.

**Comment by osro** [comment]
Here is little video about it. This is just localdev running, nothing else.
https://drive.google.com/open?id=13ZsJ9FB7HcnlPy_YnZpzweKPK9zutma5

**Comment by joshkoenig** [comment]
@osro what do you have on port `443` though? That's where Localdev "wants" to run, so it could create weirdness if something else is taking over the normal port.

@pirog this should still be predictable behavior. Maybe we can make the "alt port" configurable somehow?

**Comment by osro** [comment]
> @osro what do you have on port `443` though? That's where Localdev "wants" to run, so it could create weirdness if something else is taking over the normal port.
> 

443? 🤔 That gives me like two pages of Google Chrome instance processes.

Now after the 0.4.3 update it keeps switching on between 444 and 4433 (4443 earlier).

For these ports (444 and 4433) I have nothing running before I start the localdev. 
After the start I will get `com.docker` process running.

**Comment by westonruter** [comment]
I was surprised to see the 444 port when I got a site pulled down. I was expecting 443. Is this because I also have Lando running? Is not Localdev using Lando under the hood, and if so, shouldn't they play nice together?

**Comment by pirog** [comment]
@westonruter it's because you've got something else running on port `443` which may or may not be Lando. 

Localdev does indeed use Lando "under the hood" but the best way to conceptualize this is as Localdev using and extending Lando tech and launching as a separate and sandboxed "Lando instance" and not as Localdev as some sort of "Lando GUI". This is to say for all intents and purposes Localdev and Lando are distinct tools. With that in mind Localdev using another port when it sees `443` occupied would be a feature not, not a bug.

**Comment by osro** [comment]
Still having this issue. This happens straight if I start and stop the localdev. First the port is correct, but on second start is changes. I haven't started anything else between the start ups

**Comment by pirog** [comment]
@osro that definitely sounds more like a bug than what @westonruter is describing. We are kicking out a `0.6.0` beta release later today so if you are on the `edge` update channel you could try that and see if it resolves your issue
